client/components/domains/register-domain-step/index.jsx

```diff
@@ -109,13 +110,15 @@ const RegisterDomainStep = React.createClass( {
 		basePath: React.PropTypes.string.isRequired,
 		suggestion: React.PropTypes.string,
 		domainsWithPlansOnly: React.PropTypes.bool,
-		isSignupStep: React.PropTypes.bool
+		isSignupStep: React.PropTypes.bool,
+		searchDelay: React.PropTypes.number.isRequired
 	},
 
 	getDefaultProps: function() {
 		return {
 			onDomainsAvailabilityChange: noop,
-			analyticsSection: 'domains'
+			analyticsSection: 'domains',
+			searchDelay: parseInt( abtest( 'domainSearchDelay' ) )
```

Comment:
`searchDelay` prop is defined as required, we shouldn't need a default as it's supposed to come from the parent.